client/overview/index.js

```diff
@@ -59,6 +60,10 @@ const OverviewPage = () => {
 	const showLoanOfferError = '1' === queryParams[ 'wcpay-loan-offer-error' ];
 	const showServerLinkError =
 		'1' === queryParams[ 'wcpay-server-link-error' ];
+	const showProgressiveOnboardingEligibilityModal =
```

Comment:
I'd add a check also for `accountStatus.status !== 'pending_verification'`, otherwise we might see the modal while account is [under verification](https://github.com/Automattic/woocommerce-payments/pull/6148)

Reply:
But it's mostly for local/sandbox, in production I guess it happens very quickly so users don't notice

Reply:
Good point! Better to be sure, added in 406891f

Reply:
Awesome, thanks!